<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classifier_attr</class>
 <widget class="QWidget" name="classifier_attr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор признаков для ввода исходных данных</string>
  </property>
  <widget class="QPushButton" name="pushButtonLeft">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Доступные признаки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбранные признаки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRight_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLeft_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRight">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Left">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Right">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите признаки для ввода исходных данных</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Unavail">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Недоступные признаки</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
